From f3f45aaef50d22a122f9f763a220ddefa03a4e60 Mon Sep 17 00:00:00 2001
From: Zidan Wang <zidan.wang@freescale.com>
Date: Thu, 4 Jun 2015 16:34:28 +0800
Subject: [PATCH 1188/1594] MLK-11030-2 ARM: dts: change SAI MCLK to
 36864000HZ to support 32bit 32k format for SAI
 master mode

commit 1b105a294bf187701df4ac77e6a9e806f7e4c98d from
git://git.freescale.com/imx/linux-2.6-imx.git

Change SAI MCLK to 36864000HZ to support 32bit 32k format for SAI master
mode.

Signed-off-by: Zidan Wang <zidan.wang@freescale.com>
---
 arch/arm/boot/dts/imx7d-sdb.dts |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/boot/dts/imx7d-sdb.dts b/arch/arm/boot/dts/imx7d-sdb.dts
index 49781eb..7e8a1a5 100644
--- a/arch/arm/boot/dts/imx7d-sdb.dts
+++ b/arch/arm/boot/dts/imx7d-sdb.dts
@@ -942,7 +942,7 @@
 	assigned-clocks = <&clks IMX7D_SAI1_ROOT_SRC>,
 			  <&clks IMX7D_SAI1_ROOT_CLK>;
 	assigned-clock-parents = <&clks IMX7D_PLL_AUDIO_POST_DIV>;
-	assigned-clock-rates = <0>, <18432000>;
+	assigned-clock-rates = <0>, <36864000>;
 	status = "okay";
 };
 
-- 
1.7.5.4

